### ISSUE TITLE
Core Agent launch bug issues from 1.2.0 onwards

### ISSUE THREAD

**Issue by asgrim** [scoutapp/scout-apm-php#77]
Core Agent 1.1.11 works fine, but now 1.2.0 crashes without `--log-file /dev/null` (or a real file). I fixed #75 (the original report) with PR #76 as a workaround, but the root cause of this issue needs to be investigated and fixed properly.

```
thread 'main' panicked at 'Daemonizing failed: RedirectStreams(9)', src/main.rs:216:19
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

The issue seems to relate to the core agent requiring somewhere to write output to. For example, running the process in the foreground (i.e. omitting the `--daemonize true` parameter) works (but when the PHP process ends, so does the core agent, so not a great solution). This seems to be why always providing a `--log-file` parameter to the core agent seems to solve it, even though the output goes to `/dev/null`.

**Comment by cschneid** [comment]
Tracking in our (internal) Core Agent repo: https://github.com/scoutapp/core-agent/issues/82

**Comment by asgrim** [comment]
Looks like this is an upstream bug in a library used by core agent; 1.2.2 will include the fix for this.

**Comment by asgrim** [comment]
Fixed in 51f1fdf2ffd475eeeae08f34139ea5ea1789d956
